<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEngine</class>
 <widget class="QDialog" name="EditEngine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>New Engine</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Path to executable:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="line_name"/>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="line_path"/>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lbl_warning">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditEngine</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditEngine</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>released()</signal>
   <receiver>EditEngine</receiver>
   <slot>_select_exe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>line_path</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EditEngine</receiver>
   <slot>_path_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>line_name</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EditEngine</receiver>
   <slot>_name_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_select_exe()</slot>
  <slot>_path_changed()</slot>
  <slot>_name_changed()</slot>
 </slots>
</ui>
